ark/scripts/validate-chart-crds.sh

```
@@ -0,0 +1,185 @@
+#!/bin/bash
+#
+# validate-chart-crds.sh
+# Validates that CRDs in the Helm chart match the source CRDs in config/crd/bases/
+#
+
+set -euo pipefail
+
+# Enable verbose mode in CI for debugging
+if [ "${CI:-false}" = "true" ]; then
+    set -x
+fi
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+# Directories
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ARK_DIR="$(cd "$SCRIPT_DIR/.." && pwd)"
+CHART_DIR="$ARK_DIR/dist/chart"
+SOURCE_CRD_DIR="$ARK_DIR/config/crd/bases"
+TEMP_DIR=$(mktemp -d)
+
+# Cleanup function
+cleanup() {
+    rm -rf "$TEMP_DIR"
+}
+trap cleanup EXIT
+
+echo "Validating Helm chart CRDs match source CRDs..."
+
+# Check if helm is installed
+if ! command -v helm &> /dev/null; then
```

could just let the script fail when its missing, it'll be fairly clear from the error, we can skip the version check too just to shorten the script